code/modules/projectiles/projectile/energy/net_snare.dm

```diff
@@ -21,7 +21,7 @@
 	if(isliving(target))
 		var/turf/Tloc = get_turf(target)
 		if(!locate(/obj/effect/nettingportal) in Tloc)
-			var/obj/effect/nettingportal/NP = new (Tloc)
+			var/NP = new /obj/effect/nettingportal(Tloc)
```

Comment:
IM SORRY I TOLD YOU WRONG I TRIED TO DELETE IT BUT ITS TOO LATE

Reply:
hehe